applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/app_ZDebug.conf

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application configuration
+
+CONFIG_ML_APP_EI_DATA_FORWARDER=y
+CONFIG_ML_APP_EI_DATA_FORWARDER_BT_NUS=y
+CONFIG_ML_APP_EI_DATA_FORWARDER_SENSOR_EVENT_DESCR="accel_xyz"
+
+CONFIG_ML_APP_LED_STATE=y
+CONFIG_ML_APP_LED_STATE_VALUE_THRESH=700
+CONFIG_ML_APP_LED_STATE_ANOMALY_THRESH=300
+
+CONFIG_ML_APP_ML_RUNNER=y
+CONFIG_ML_APP_ML_RUNNER_SENSOR_EVENT_DESCR="accel_xyz"
+CONFIG_ML_APP_ML_RUNNER_WINDOW_SHIFT=0
+CONFIG_ML_APP_ML_RUNNER_FRAME_SHIFT=50
+
+CONFIG_ML_APP_ML_STATE=y
+CONFIG_ML_APP_ML_STATE_CONTROL_BUTTON_ID=0x0000
+
+CONFIG_CAF=y
+
+CONFIG_CAF_INIT_LOG_SENSOR_EVENTS=n
+
+CONFIG_CAF_SENSOR_SAMPLER=y
+
+CONFIG_CAF_BUTTONS=y
+CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
+
+CONFIG_CAF_CLICK_DETECTOR=y
+
+CONFIG_CAF_BLE_STATE=y
+CONFIG_CAF_BLE_USE_LLPM=n
+
+CONFIG_CAF_BLE_ADV=y
+CONFIG_CAF_BLE_ADV_FAST_ADV=y
+
+CONFIG_CAF_LEDS=y
+CONFIG_CAF_LEDS_GPIO=y
+
+################################################################################
+# EI wrapper configuration
+
+CONFIG_CPLUSPLUS=y
+CONFIG_LIB_CPLUSPLUS=y
+CONFIG_STD_CPP11=y
+CONFIG_FPU=y
+
+# Use the NCS machine learning model for acceleration readouts coming from HW accelerometer
+CONFIG_EDGE_IMPULSE=y
+CONFIG_EDGE_IMPULSE_URI="https://studio.edgeimpulse.com/v1/api/24687/deployment/download?type=zip"
+CONFIG_EI_WRAPPER=y
+CONFIG_EI_WRAPPER_DATA_BUF_SIZE=1000
+
+################################################################################
+# System configuration
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_ISR_STACK_SIZE=1536
+CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_IDLE_STACK_SIZE=512
+CONFIG_BT_RX_STACK_SIZE=1600
+CONFIG_BT_HCI_ECC_STACK_SIZE=1300
+
+CONFIG_HEAP_MEM_POOL_SIZE=2048
+
+CONFIG_SPEED_OPTIMIZATIONS=y
+CONFIG_HW_STACK_PROTECTION=y
+
+CONFIG_BOOT_BANNER=n
+CONFIG_NUM_COOP_PRIORITIES=10
+CONFIG_NUM_PREEMPT_PRIORITIES=11
+
+CONFIG_EVENT_MANAGER=y
+CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
+CONFIG_REBOOT=y
+
+CONFIG_GPIO=y
+
+CONFIG_LED=y
+CONFIG_LED_GPIO=y
+
+# Using built-in sensor
+CONFIG_SENSOR=y
+CONFIG_ADXL362=y
+CONFIG_ADXL362_ACCEL_RANGE_8G=y
+CONFIG_ADXL362_ACCEL_ODR_200=y
```

Comment:
100 Hz may be sufficient in our case (we sample every 20 ms - 50Hz)

Reply:
leftover, right